inc/saiport.h

```diff
@@ -364,6 +367,16 @@ typedef enum _sai_port_connector_failover_mode_t
     SAI_PORT_CONNECTOR_FAILOVER_MODE_SECONDARY
 } sai_port_connector_failover_mode_t;
 
+/**
+ * @brief Attribute data for #SAI_PORT_ATTR_RECYCLE_PORT_HEADER_MODE
+ */
+typedef enum _sai_port_recycle_port_header_mode_t
+{
+    /** Ethernet header mode */
+    SAI_PORT_RECYCLE_PORT_HEADER_MODE_ETH,
```

Comment:
This additional header_mode for recycle port probably needs more reviews/discussion.  I'm not sure if it's needed now.